usage-reporter/test/worker.test.js

```diff
@@ -0,0 +1,318 @@
+import { describe, it, expect, beforeEach, afterEach, vi } from 'vitest'
+import { env } from 'cloudflare:test'
+import { epochToTimestampMs } from '../lib/epoch.js'
+import worker from '../bin/usage-reporter.js'
+import filbeamAbi from '../lib/FilBeamOperator.abi.json'
+import {
+  withDataSet,
+  withRetrievalLog,
+  EPOCH_95_TIMESTAMP_ISO,
+  EPOCH_98_TIMESTAMP_ISO,
+  EPOCH_99_TIMESTAMP_ISO,
+  EPOCH_100_TIMESTAMP_ISO,
+} from './test-helpers.js'
+
+describe('usage reporter worker scheduled entrypoint', () => {
+  let simulateContractCalls
+  let writeContractCalls
```

Comment:
Please add some typings here. I think `/** @type {object[]} */` can be a minimal viable description.